### DISCUSSION TITLE
Working with vue 2 files is became a nightmare

### DISCUSSION THREAD

**Discussion by goof03** [LazyVim/LazyVim#3971] [Q&A]
I work on an old vue 2 project with migration to composition API.
I use eslint 8.47 with eslint-plugin-prettier in this project. 
On lspinfo i have volar eslint and vtsls and the eslint for prettier plugin recipe is in my conf

everything was working fine before … And yarn eslint --fix do the job correctly

But since some updates i get some config file no found error, each time i save i get a new wrong format, sometime with syntax error sometime lsp crash … inlay hints are unreadable seems to add some typescript definition in my js code …

How to go back to my stable editor please ? 
I have a real job to feed my family… I would be happy to be a beta tester if I had the time

how to make LazyVim great again please ?

**Comment by iamxiaojianzheng** [comment]
Can you show me a screenshot of the exception?

**Reply by goof03** [reply]
It's magical seems to work this morning …

but this the kind of messagesi get this evening:
lazyvim eslint: -32603: Request textDocument/diagnostic failed with message: Could not find config file

[ERROR][2024-07-10 08:47:20] ...lsp/handlers.lua:623> "[Vue Named Pipe Client] Timeout" 

[ERROR][2024-07-09 02:42:07] ...lsp/handlers.lua:623    'Cannot read config file: /home/zorglub/…/frontend/.eslintrc.js
        │  Error: require() of ES Module /home/zorglub/…/frontend/.eslintrc.js from /home/zorglub/Projets/Pro/Myotis/web/vision-s
        │ tack/frontend/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported. .eslintrc.js is treated as an ES module file as it is a .js file whose neare
        │ st parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules. Instead either rename .eslintrc.js to
        │ end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonj
        │ s" in /home/zorglub/…/frontend/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
        │

**Comment by goof03** [comment]
Doh I found the bug...

You can't open files in multiple projects (with different .eslint.js and or eslint version) 
if you do seems the last eslint conifg is used …

(i work on vue frontend and a node backend in the same time)
